core/trino-spi/src/main/java/io/trino/spi/security/ViewExpression.java

```diff
@@ -38,8 +44,12 @@ public ViewExpression(String identity, Optional<String> catalog, Optional<String
         }
     }
 
+    /**
+     * @return user as whom the view expression will be evaluated. If empty identity is returned
+     * then session user is used.
+     */
     @Internal
-    public String getIdentity()
+    public Optional<String> getIdentity()
```

Comment:
Sometimes we have to introduce breaking changes.
So far we haven't been ultra strict about that, so I think it's fine to just change this method and add exclusion to revapi check. Let's defer intro of `@Internal` until we have more clairty what we want to achieve.